tamr_unify_client/base_collection.py

```diff
@@ -102,6 +102,19 @@ def by_external_id(self, resource_class, external_id):
 
         return items[0]
 
+    def delete_by_resource_id(self, resource_id):
+        """
+        Deletes a collection's resource by resource_id.
+
+        :param resource_id: the resource_id of the resource that will be deleted.
+        :type: str
+        :return: status code
+        :rtype: HTTP status code
```

Comment:
What you're returning isn't the status code but the response itself, which has type `requests.Response`